tests/pdarray_test.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+import arkouda as ak
+import numpy as np
+import subprocess, os
+
+class PdArrayTest(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(cls):
+        try: 
+            arkouda_path = os.environ['ARKOUDA_SERVER_PATH']
+        except KeyError:
+            raise EnvironmentError(('The ARKOUDA_SERVER_PATH env variable output from the ' + 
+                                   '"which arkouda_server" command must be set'))
+        
+        PdArrayTest.ak_server = subprocess.Popen([arkouda_path, '--ServerPort=5566', '--quiet'])
+    
+    def setUp(self):
+        ak.client.connect(port=5566)
+      
+    def testPdArrayAddInt(self):
```

Comment:
@hokiegeek2 please add test cases to `tests/operator_tests.py` to check for the numpy scalar cases

Reply:
@mhmerrill sure thing! on it now.